src/scripts/todolist.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line no-restricted-globals
```

Comment:
[OPTIONAL] Please consider fixing linters errors instead of disabling them as they help us to write clean codes and follow best coding practices.